yelp_package/dockerfiles/xenial/Dockerfile

```diff
@@ -15,7 +15,7 @@
 ARG DOCKER_REGISTRY=docker-dev.yelpcorp.com/
 FROM ${DOCKER_REGISTRY}xenial_pkgbuild
 
-ARG PIP_INDEX_URL=https://pypi.yelpcorp.com/simple
+ARG PIP_INDEX_URL=https://pypi.yelpcorp.com/xenial/simple
```

Comment:
are these still needed (here and elsewhere)?

Reply:
Re-read the description and I see, this is needed and intentionally here (though I'm not 100% clear why)

Reply:
I think it may not be needed on this one since it uses our internal pkgbuild image, but for most of these it is still needed due to using the public Ubuntu base images. (Even though they all pull from our registry, they're pulling mirrored Ubuntu images as opposed to our internal modified Ubuntu images.)

In the near future (hopefully this quarter) we should be able to consolidate on the main root registry again (and without requiring pip-custom-platform) so we can do a followup round later to clean all of these up even more.